BusinessPass.java

```diff
@@ -0,0 +1,22 @@
+public class BusinessPass extends Pass{
+private int loyaltyPoints;
+    public BusinessPass(int passId, String name, int luxury, int credit,int loyaltyPoints){
```

Comment:
Credit and loyaltyPoints shouldn't be in the constructor since the specification says `are created with a specified luxury rating, 30 credits and 20 loyalty points `